### ISSUE TITLE
Entering a GPS coordinate in the search box no longer works

### ISSUE THREAD

**Issue by downtownallday** [nextcloud/maps#802]
prior to v0.2.0 I could enter GPS coordinates in the search box and be able to zoom to that point. eg "44.103629,-71.352803". It doesn't seem to work any longer.

**Comment by aaronsegura** [comment]
+1 Same for me.  This is a feature I use all the time.

**Comment by aaronsegura** [comment]
This is still an issue for me after updating to 0.2.1:

![image](https://user-images.githubusercontent.com/1391955/188737666-fbd2e5c2-d230-420f-87c1-1d625dd3aadb.png)

I was searching for `29.518623, -98.393236`

Also, the hamburger icon now blocks part of the search box.

**Comment by tacruc** [comment]
Ok, the Issue is the blank space after the ",".
Without that it works, for me. Can you confirm this?

**Comment by tacruc** [comment]
Can you check if #851  solves your issue?

**Comment by aaronsegura** [comment]
I can verify that searching on v0.2.1 without a space after the comma works, but I'm unable to verify the PR.  After attempting to clone the repo, checking out the new branch, and running into all sorts of issues trying to get it to work, I finally decided to make the single-line edit to my live `src/components/map/SearchField.vue` file as described in the PR, but it doesn't seem to have any effect:

    $ sed -n 168p src/components/map/SearchField.vue
                            const regResult = coordinateRegExp.exec(searchQuery.replace(/\s*/g, ''))

Any suggestions?

**Comment by tacruc** [comment]
Yes the vue code is "compiled" into some more compact JS file. (I think its called maps.js). Probably this is the proccess you ran into errors when trying to checkout the code.
